.gitignore

```diff
@@ -1,168 +1,22 @@
-# Ignore created packages
-/package
-
-/.idea
```

Comment:
Why?

Reply:
This is for developers with PyCharm.

Reply:
I have reverted that deletion. 
https://github.com/ynput/example-studio-addon/pull/3/commits/7c00197b79cfc43dbee52e9922f0aa486ec2a6bd